server/chat/background/task.py

```
@@ -650,14 +661,20 @@ def run_background_chat(
             
-            # Regenerate incident summary now that RCA chat has completed
+            # Regenerate incident summary (skip if already enqueued by inner function)
             try:
-                from chat.background.summarization import generate_incident_summary_from_chat
-                generate_incident_summary_from_chat.delay(
-                    incident_id=incident_id,
-                    user_id=user_id,
-                    session_id=session_id,
-                )
+                with db_pool.get_admin_connection() as conn:
+                    with conn.cursor() as cursor:
```

Ruff (0.15.13)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)